website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,7 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+        window.document.cookie.split(";").forEach(function(c) { window.document.cookie = c.replace(/^ +/, "").replace(/=.*/, "=;expires=" + new Date().toUTCString() + ";path=/"); });
```

Comment:
Can you provide documentation about where you got this snippet? Overwriting `window.focument.cookie` over and over again seems wrong to me.